src/mccabe/checks.rs

```diff
@@ -0,0 +1,69 @@
+use rustpython_ast::{ExcepthandlerKind, ExprKind, Stmt, StmtKind};
+
+use crate::ast::types::Range;
+use crate::check_ast::Checker;
+use crate::checks::{Check, CheckKind};
+
+fn get_complexity_number(stmts: &[Stmt]) -> isize {
```

Comment:
I wonder if we should memoize this? E.g., in theory, for nested functions, we're going to first run this check for the outer function (which will include computing complexity for the inner function), then we're going to run it again when we hit the outer function in the AST visitor. For deeply nested functions, that could really add up. Can we reduce the number of re-iterations?